core/zcncrypto/ed255190chain.go

```
@@ -57,6 +57,11 @@ func (ed *ED255190chainScheme) GenerateKeys() (*Wallet, error) {
 	return w, nil
 }
 
+//GenerateKeysWithEth - not implemented
+func (ed *ED255190chainScheme) GenerateKeysWithEth(mnemonic, password string) (*Wallet, error) {
+	return nil, nil
```

I would return an error so not to confuse the client caller